docs/wes-qc-hail.md

```diff
@@ -114,31 +114,54 @@ To start a new task via `hlrun_remote`, first end the existing tmux session, if
 ### 0. Resource Preparation
 All steps in this section need to be run only once before your first run. It prepares the reference dataset for the subsequent steps.
 
-1. Create the 1000G population prediction resource set.
+1. **Create the 1000G population prediction resource set**.
+
 This resource set is required for the super-population prediction on the population PCA step.
 Then you can reuse it with any data cohort.
 
 ```shell
 spark-submit 0-resource_preparation/1-import_1kg.py --all
 ```
 
-### 1. Load data from VCFs into Hail
+### 1. Load data
 
-1. Load VCFs into Hail and save as a Hail MatrixTable
+1. **Load VCFs into Hail and save as a Hail MatrixTable**
 
 ```shell
 spark-submit 1-import_data/1-import_gatk_vcfs_to_hail.py
 ```
 
+2. **Annotate metadata**
+
+This script annotates samples with all provided metadata:
+VerifyBamId Freemix score, self-reported sex, self-reported ethnicity, etc.
+
+If you don't have some (or even any) of these annotations,
+put `null` instead of the corresponding filename in the config file.
```

Comment:
specify the key to put it as `null` i.e. `verifybamid_selfsm`

Reply:
Added.